src/EETH.sol

```diff
@@ -15,12 +15,18 @@ import "./interfaces/ILiquidityPool.sol";
 
 contract EETH is IERC20Upgradeable, UUPSUpgradeable, OwnableUpgradeable, IERC20PermitUpgradeable, IeETH {
     using CountersUpgradeable for CountersUpgradeable.Counter;
+    //--------------------------------------------------------------------------------------
+    //---------------------------------  STATE-VARIABLES  ----------------------------------
+    //--------------------------------------------------------------------------------------
+    
     ILiquidityPool public liquidityPool;
 
     uint256 public totalShares;
     mapping (address => uint256) public shares;
     mapping (address => mapping (address => uint256)) public allowances;
     mapping (address => CountersUpgradeable.Counter) private _nonces;
+    mapping (address => bool) public whitelistedSpender;
```

Comment:
@seongyun-ko, Could you confirm my reasoning here:

Becausing constant and immutable variables are stored directly in the bytecode, I can place the new mapping above these values without having to worry about a storage collision

Reply:
ho, sounds right :) you can always verify it by checking the storage layout of this contract before/after your update

Reply:
Verified ✅